<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>20190084</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_watch">
     <attribute name="title">
      <string>Watch</string>
     </attribute>
     <widget class="QLabel" name="clock_label_time">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>801</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="clock_label_date">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>801</width>
        <height>57</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_stopwatch">
     <attribute name="title">
      <string>Stopwatch</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>801</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="stopwatch_btn_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopwatch_btn_stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopwatch_btn_reset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="stopwatch_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>801</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>00 : 00.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_timer">
     <attribute name="title">
      <string>Timer</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>801</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="timer_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="timer_stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="timer_reset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="timer_btn_hour_up">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="timer_btn_sec_up">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>180</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="timer_btn_minute_up">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>180</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="timer_btn_hour_down">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="timer_btn_sec_down">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>260</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="timer_btn_minute_down">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>801</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="timer_label_hour">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="text">
          <string>:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timer_label_minute">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="text">
          <string>:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timer_label_sec">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
